common/saber_base.h

```diff
@@ -46,6 +46,7 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(OFF_CLASH)                    \
     DEFINE_EFFECT(NEXT_QUOTE)                   \
     DEFINE_EFFECT(INTERACTIVE_PREON)            \
+    DEFINE_EFFECT(INTERACTIVE_BLAST)            \
```

Comment:
Since this is obviously different from EFFECT_FIRE in some way, please add a comment that explains what that difference is.

Reply:
added